### PR TITLE
Portfolio menu

### PR DESCRIPTION
**Pull request by stevenmukama**
Added styles in styles.css.
Added changes in index.html.
Added changes to match the design in Figma.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+          <div class="btn-line"></div>
+          <div class="btn-line"></div>
+      </div>
+        <ul class="itemslist" id="elements">
+            <li class="listitem"><a href="#portfolio">Portfolio</a></li>
+            <hr class="spaceitem" />
+            <li class="listitem" ><a href="#about">About</a></li>
+            <hr class="spaceitem"/>
+            <li class="listitem" ><a href="#contact">Contact</a></li>
+            <hr class="spaceitem"/>
```
Comment (tufoinnkuo10) [inline index.html:29]: - [ ] Great job. please ensure the menu links take you to the correct page sections. the `about `and `contact `links are redirecting to wrong page sections. please fix that.

File: main.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
+const menuBtn = document.querySelector('.navbarmenu');
+const menuItems = document.querySelector('.itemslist');
+const listItems = document.querySelectorAll('.listitem');
+const navBar = document.querySelector('.navbar');
+const navbarIcon = document.querySelector('.navbaricon');
+const crossButton = document.querySelectorAll('.btn-line');
+
```
Comment (tufoinnkuo10) [inline main.js:7]: - [ ] Great work. you are missing Eslint checks. ESLint is a tool for identifying and reporting on patterns found in your JavaScript code, with the goal of making it more consistent and avoiding bugs. you can check how to add it [here](https://github.com/microverseinc/linters-config/tree/master/javascript).

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @stevenmukama 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Mobile menu hamburger works as expected
✅ Menu close icon works perfectly well

Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by ichala — APPROVED** [review]
Hi @stevenmukama  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-  Changes Implemenetd✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**